finance_manager/settings/base.py

```diff
@@ -36,6 +36,7 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
 
+    'crispy_forms',
```

Comment:
It would be great to divide apps for installed and local.
```
'django.contrib.messages',
'django.contrib.staticfiles',
'crispy_forms',

'users',
```